chef/cookbooks/crowbar/templates/default/crowbar-upgrade-os.sh.erb

```diff
@@ -79,6 +79,13 @@ initiate_node_upgrade()
     # Upgrade the distribution non-interactively
     log "Executing zypper ref"
     zypper --no-color --releasever <%= @target_platform_version %> ref -f
+
+    # check to see if openvswitch is gonna be updated
+    is_openvswitch_updated=`zypper --no-color --non-interactive list-updates|grep -q openvswitch`
```

Comment:
The -q switch here only makes it quiet. So the return value would be 0 on success.  So this will always put nothing (as it returns nothing the stdout) into the is_openvswitch_updated wouldn't it? which would mean the renabling triggered by the 'if' on line 148 would never be be triggered. At least in my understanding.

Reply:
@matthewoliver this was followed with https://github.com/crowbar/crowbar-core/pull/1558 (which we believe might not be needed after all because we might have package already...)